### PR TITLE
Update CommandAPI to 9.4.0

### PR DESCRIPTION
**Pull request by Nobody314159**

### CODE REVIEW

File: src/main/java/com/playmonumenta/structures/commands/ActivateSpecialStructure.java [hunk 1]
```diff
@@ -17,20 +20,15 @@ public static void register(Plugin plugin) {
...
-			.register();
+		Argument<String> labelArg = new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES);
+		TextArgument pathArg = new TextArgument("special_structure_path");
 
 		new CommandAPICommand(command)
 			.withPermission(perms)
-			.withArguments(new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES))
-			.withArguments(new TextArgument("special_structure_path"))
+			.withArguments(labelArg)
+			.withOptionalArguments(pathArg)
```
Comment (dmdinnc) [inline src/main/java/com/playmonumenta/structures/commands/ActivateSpecialStructure.java:29]: Should this be optional? it's nullable in there, but in implementation it's checking if null or empty and forcing failure

File: src/main/java/com/playmonumenta/structures/commands/ListRespawningStructures.java [hunk 2]
```diff
@@ -13,18 +16,13 @@ public static void register() {
...
-			.executes((sender, args) -> {
-				list(sender, null);
-			})
-			.register();
+		Argument<String> labelArg = new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES);
 
 		new CommandAPICommand(command)
 			.withPermission(perms)
-			.withArguments(new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES))
+			.withOptionalArguments(labelArg)
```
Comment (dmdinnc) [inline src/main/java/com/playmonumenta/structures/commands/ListRespawningStructures.java:23]: Same here, if we're checking null later, we shouldn't leave it optional

Comment (Nobody314159) [reply src/main/java/com/playmonumenta/structures/commands/ListRespawningStructures.java:23]: This one should be optional - if it is null it lists all structures

File: src/main/java/com/playmonumenta/structures/commands/ActivateSpecialStructure.java (hunk 1 above)
Comment (Nobody314159) [reply src/main/java/com/playmonumenta/structures/commands/ActivateSpecialStructure.java:29]: Yeah, looks like it. Not sure why it was written before to have a command that always failed... Fixing

**Pull request merged**